Modules/System Tests/Image/src/ImageTests.Codeunit.al

```diff
@@ -159,6 +181,32 @@ codeunit 135135 "Image Tests"
         Assert.ExpectedError('Parameter Width must be greater than 0');
     end;
 
+    [Test]
+    procedure RotateFlipTest()
+    var
+        TempBlob: Codeunit "Temp Blob";
+        RotateFlipType: Enum "Rotate Flip Type";
+        OutStream: OutStream;
+        ImageAsBase64HorizontalTxt: Label 'iVBORw0KGgoAAAANSUhEUgAAAAoAAAAFCAIAAADzBuo/AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAAUSURBVBhXY6AS+P//PxqDQsDAAADP2QX7LebCcQAAAABJRU5ErkJggg==', Locked = true;
+        ImageAsBase64VerticalTxt: Label 'iVBORw0KGgoAAAANSUhEUgAAAAUAAAAKCAIAAADzWwNnAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAAXSURBVBhXYyAP/P//H8oiDpCqHgoYGADezwX75D+gjQAAAABJRU5ErkJggg==', Locked = true;
+    begin
+        // [Given] base64 encoded data, create image
+        Image.FromBase64(ImageAsBase64HorizontalTxt);
+
+        // [When] rotate image
+        Image.RotateFlip(RotateFlipType::Rotate90FlipNone);
```

Comment:
Add a test that rotates an image 4 times and verifies the new image is the same ;)

Reply:
Added RotateFlip360Test().